src/pages/SubscriptionService.tsx

```diff
@@ -135,105 +134,109 @@ class SubscriptionService extends Component<Props, SubscriptionServiceState> {
+                  name="department_name"
+                  type="text"
+                  defaultValue={service.department_name}
+                  onChange={this.handleInputChange}
+                  className="mb-4"
+                />
 
-              <label className="m-0">{t("organization_fiscal_code")}*</label>
-              <input
-                name="organization_fiscal_code"
-                type="text"
-                defaultValue={service.organization_fiscal_code}
-                onChange={this.handleInputChange}
-                className="mb-4"
-              />
+                <label className="m-0">{t("organization")}*</label>
+                <input
+                  name="organization_name"
+                  type="text"
+                  defaultValue={service.organization_name}
+                  onChange={this.handleInputChange}
+                  className="mb-4"
+                />
 
-              {storage.isApiAdmin && (
-                <div>
-                  <label className="m-0">
-                    {t("max_allowed_payment_amount")}
-                  </label>
-                  <input
-                    name="max_allowed_payment_amount"
-                    type="text"
-                    defaultValue={
-                      service.max_allowed_payment_amount
-                        ? service.max_allowed_payment_amount.toString()
-                        : undefined
-                    }
-                    onChange={this.handleInputChange}
-                    className="mb-4"
-                  />
-                </div>
-              )}
+                <label className="m-0">{t("organization_fiscal_code")}*</label>
+                <input
+                  name="organization_fiscal_code"
+                  type="text"
+                  defaultValue={service.organization_fiscal_code}
+                  onChange={this.handleInputChange}
+                  className="mb-4"
+                />
 
-              {storage.isApiAdmin && (
-                <div>
-                  <label className="m-0">{t("authorized_ips")}</label>
-                  <input
-                    name="authorized_cidrs"
-                    type="text"
-                    defaultValue={service.authorized_cidrs.join(";")}
-                    onChange={this.handleInputChange}
-                    className="mb-4"
-                  />
-                </div>
-              )}
+                {storage.isApiAdmin && (
+                  <div>
+                    <label className="m-0">
+                      {t("max_allowed_payment_amount")}
+                    </label>
+                    <input
+                      name="max_allowed_payment_amount"
+                      type="text"
+                      defaultValue={
+                        service.max_allowed_payment_amount
+                          ? service.max_allowed_payment_amount.toString()
+                          : undefined
```

Comment:
If `max_allowed_payment_amount === 0` you are assigning `undefined`. Is that correct?

Reply:
It's an old condition